<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moduleDialog</class>
 <widget class="QDialog" name="moduleDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>205</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>462</width>
    <height>205</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Module settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Module name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="dbLbl">
     <property name="text">
      <string>Redis DB index</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="dbEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="execLbl">
     <property name="text">
      <string>Execution path</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="execEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dirLbl">
     <property name="text">
      <string>Working dir</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="LineEditWithButton" name="dirEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="parLbl">
     <property name="text">
      <string>App params</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="LineEditWithButton" name="parEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Records per logger</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="rplEdit">
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithButton</class>
   <extends>QLineEdit</extends>
   <header location="global">lineeditwithbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
